runtime/src/gpu/nvidia/gpu-nvidia.c

```diff
@@ -39,12 +39,20 @@
 #include <assert.h>
 #include <stdbool.h>
 
+#include "hwloc.h"
+#include "hwloc/cuda.h"
+#include "chpl-topo.h"
+
```

Comment:
What happens if a user does `CHPL_HWLOC=none`?

Reply:
Also, we seem to avoid directly including "hwloc.h", why is it needed here? Can all of the hwloc stuff stay in `topo/hwloc`?

Reply:
I often forget that `hwloc` is optional. Fixed.